src/validation/transaction.cairo

```diff
@@ -79,6 +101,40 @@ pub fn validate_transaction(
     Result::Ok(tx_fee)
 }
 
+pub fn is_final_tx(tx: @Transaction, block_height: u32, block_time: u32) -> Result<(), ByteArray> {
+    if *tx.lock_time < LOCKTIME_THRESHOLD && *tx.lock_time < block_height {
```

Comment:
Ahaha, this is cheat :) But correct, in most of the cases locktime is zero so we will exit early on this check

Reply:
hi, seems quite similar to the cpp one, not sure what is "cheat" here :)

Reply:
@m-kus why do you think it is a cheat? It looks similar to cpp version indeed.